addons/kubearmor/enable

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+
+config=$(sudo microk8s config)
+
+echo "$config" > $SNAP_DATA/.kube/config
+
+export KUBECONFIG=$SNAP_DATA/.kube/config
+
+if hash karmor 2>/dev/null; then
+    echo "karmor tool is already installed"
+else 
+    curl -sfL http://get.kubearmor.io/ | sudo sh -s -- -b /usr/local/bin
```

Comment:
It would be best if you placed the karmor binary (or a karmor wrapper that would include `export KUBECONFIG=$SNAP_DATA/credentials/client.config`) under `"$SNAP_COMMON/plugins"`. This way the user would be able to call `microk8s karmor` directly. See:
https://github.com/canonical/microk8s-core-addons/blob/main/addons/cis-hardening/enable#L301